src/tests/suite/daos_test.c

```diff
@@ -30,8 +30,8 @@
 #include "daos_test.h"
 
 /** All tests in default order (tests that kill nodes must be last) */
-static const char *all_tests = "mpceXViADKCoROdr";
-static const char *all_tests_defined = "mpceXVixADKCoROdr";
+static const char *all_tests = "mpceXViABDKCoROdr";
```

Comment:
so if someone runs daos_test manually, will this test run?
if we know it doesn't work then it shouldn't.

Reply:
Presently, yes. If someone runs manually, it will run the test and will fail. I thought it will be useful for developers to traige the issue if we land it... Anyway, we can keep this PR open and ask the developers to use this code for triaging the issues.

Reply:
we shouldn't land any tests that fail by default.
IMO this should be a reproducer for tickets that you create for developers to look into. btw the ticket you created is with the default pool size, so as i mentioned above, you should try with large pool size as is needed by the test and see what are the issues and create tickets accordingly. -1007 is not a bug, just not enough space for the test to run.

Reply:
Agreed. I will update the setup to create a large pool. But, still daos_server crashing is not a good sign. Someone needs to investigate why daos_server crashes.

Reply:
indeed, please make that clear in that ticket then that the server crash is the issue.
i suspect it has to do with OOM, but not sure